foxpuppet/windows/browser/window.py

```diff
@@ -32,6 +32,14 @@ def notification(self):
         except NoSuchElementException:
             return None  # no notification is displayed
 
+    @property
+    def tabs(self):
```

Comment:
There are a couple of other items in the tabbar. There are not only the tabs. Eg. pinned tabs, the arrows.